app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def create_a_file_from_input():
```

Comment:
If you test it you'll see that all lines are appended to the same line
Lines written by a user should be separated

Reply:
```
Please, enter the file name: filename
Please, enter new line of content: first
Please, enter new line of content: second
```
filename.txt:
```
firstsecond
```
instead of 

```
first
second
```